capstone/src/main/webapp/youtubeTypescript.ts

```diff
@@ -406,3 +410,31 @@ function formatCommentForSpreadsheet(comment: string) {
   formattedComment = formattedComment.replace(/\s+/g, " ");
   return formattedComment;    
 }
+
+/** Creates chart of analyzed comments and requested attributes*/
+function drawTableChart() {      
+  const requestedAttributes = getRequestedAttributes();
+  let tableData = new google.visualization.DataTable();
+  // Add columns
+  tableData.addColumn('string', 'COMMENT');
+  for (let i = 0; i < requestedAttributes.length; i++) {
+    tableData.addColumn('number', requestedAttributes[i]);
+  }
+  // Add rows
+  tableData.addRows(ANALYZED_COMMENTS.length);
+  for (let i = 0; i < ANALYZED_COMMENTS.length; i++) {
+    tableData.setCell(i, 0, ANALYZED_COMMENTS[i])
+    for (let j = 1; j < ATTRIBUTE_DATA[i].length + 1; j++) {
+      tableData.setCell(i, j, ATTRIBUTE_DATA[i][j-1])
+    }
+  }
+  let table = new google.visualization.Table(document.getElementById('table-container'));
+  let formatter = new google.visualization.ColorFormat();
+  formatter.addRange(0, .2, 'white', '#6B8E23');
+  formatter.addRange(.2, .8, 'white', '#ffd800');
+  formatter.addRange(.8, 1, 'white', '#DC143C');
```

Comment:
Just a note, I will be changing these colors to the scale on https://support.perspectiveapi.com/s/about-the-api-best-practices-risks for the text analyzer's bar graph (I think all other bar graphs on the site use the same code)

Reply:
addressed in "address chris nits"